Last-Update: 2012-08-13
Author: "Ying-Chun Liu (PaulLiu)" <paulliu@debian.org>
Forwarded: 2012-08-13
Bug-Debian: http://bugs.debian.org/681060
Description: fix crash on unicode character(s) paste.
 However "Paste Unformatted" is still affected by the same issue.

--- a/src/af/xap/xp/xap_EncodingManager.cpp
+++ b/src/af/xap/xp/xap_EncodingManager.cpp
@@ -1260,10 +1260,10 @@
 		"UTF-16-BE",		// my guess
 		0 };
 	static const char * (szUCS2LENames[]) = {
+		"UTF-16LE",			// superset
 		"UCS-2LE",			// preferred
 		"UCS-2-LE",			// older libiconv
 		"UNICODELITTLE",	// older glibc
-		"UTF-16LE",			// superset
 		"UTF-16-LE",		// my guess
 		0 };
 
